services/README.md

```diff
@@ -0,0 +1,51 @@
+# Guidelines for using self-hosted services in MetaBrainz projects
+
+This is a set of guidelines for the main contributors of MetaBrainz
+projects to keep their use of self-hosted services maintainable.
+
+This document mentions general guidelines only about **documentation
+requirements** for a project when using of a self-hosted service so
+that system adminstrators can actually understand what to do or to
+avoid without having to read the code to understand an emergency
+situation or to necessarily rely on the code authors to take action.
+
+It is completed by the service-specific guidelines in this directory
+which rather focus on **implementation requirements**.
+
+See also the private system administration wiki for details about how
+these self-hosted services are hosted in the MetaBrainz infrastructure.
+
+The below lists are indicative of what should likely be documented.
+Those are not exhaustive.
+
+## Requirements to be documented
+
+- Tolerance to connectivity issues:
+  What are the consequences of connectivity issues for the project?
+- Maintenance mode:
+  Is there a maintenance mode that can be set for maintenance of the
+  self-hosted service the project depeds on?
```

Comment:
depeds -> depends

Reply:
Thanks, fixed in https://github.com/metabrainz/guidelines/pull/17/commits/e1d5fcb5665cc986dafb4357e47348cd5850c3f5 too